specs/interop/supervisor.md

```diff
@@ -234,6 +261,16 @@ Parameters: (none)
 
 Returns: `SupervisorSyncStatus`
 
+#### `supervisor_dependencySet`
+
+Retrieves the full dependency set configuration,
+to determine which chains are available at any particular block.
+The returned configuration may change with new chain upgrades. 
+
+Parameters: (none)
```

Comment:
I would have instinctively though this should take a block number as parameter ..